subgraphs/aave-forks/src/mapping.ts

```diff
@@ -95,39 +124,82 @@ export function _handleReserveInitialized(
   protocolData: ProtocolData,
   stableDebtToken: Address = Address.fromString(ZERO_ADDRESS)
 ): void {
-  // create tokens
-  const outputTokenEntity = getOrCreateToken(outputToken);
-  const variableDebtTokenEntity = getOrCreateToken(variableDebtToken);
-
-  // update and initialize specofic market variables
-  const market = getOrCreateMarket(underlyingToken, protocolData);
-
-  market.name = outputTokenEntity.name;
-  market.outputToken = outputTokenEntity.id;
-  market.createdBlockNumber = event.block.number;
-  market.createdTimestamp = event.block.timestamp;
-  market._vToken = variableDebtTokenEntity.id;
+  // create VToken template
   VTokenTemplate.create(variableDebtToken);
-  if (stableDebtToken != Address.fromString(ZERO_ADDRESS)) {
-    market._sToken = getOrCreateToken(stableDebtToken).id;
+  // create AToken template to watch Transfer
+  ATokenTemplate.create(outputToken);
+
+  const manager = new DataManager(
+    outputToken,
+    underlyingToken,
+    event,
+    protocolData
+  );
+  const market = manager.getMarket();
+  const outputTokenManager = new TokenManager(outputToken, event);
+  const vDebtTokenManager = new TokenManager(
+    variableDebtToken,
+    event,
+    TokenType.NON_REBASING
```

Comment:
aTokens, variableDebtTokens, and stableDebtTokens are REBASING

Reply:
fixed